src/pages/Map.js

```diff
@@ -1,12 +1,12 @@
 import React, { useState, useEffect } from "react";
 import { 
   GoogleMap, 
-  useLoadScript, 
-  Marker, 
-  InfoWindow 
+  useLoadScript,
+  MarkerClusterer
 } from '@react-google-maps/api';
 import { mapStyles } from '../styling/mapStyles';
 import { getAllVerified } from '../util';
+import CustomMarker from '../components/CustomMarker';
 
```

Comment:
organize imports lol sorry I know this is annoying but they would enforce it like crazy at my internship